cmd/access/node_builder/access_node_builder.go

```diff
@@ -375,6 +409,111 @@ func (builder *FlowAccessNodeBuilder) BuildConsensusFollower() AccessNodeBuilder
 	return builder
 }
 
+func (builder *FlowAccessNodeBuilder) BuildExecutionDataRequester() *FlowAccessNodeBuilder {
+	var ds *badger.Datastore
+	var bs network.BlobService
+	var processedBlockHeight *storage.ConsumerProgress
```

Comment:
Leo added a readme for the jobqueue (in PR now): https://github.com/onflow/flow-go/pull/2373/files#diff-066d544a40048fbcccc0b571ac963951665dd711c88b27d496b9f082a05fb0e8R47

it manages/stores the jobqueue state, specifically the `lastProcessedIndex` value

Reply:
I guess what I'm confused about is why only the ConsumerProgress part is in the `storage` package, while everything else is in the `jobqueue` package?

Reply:
I'm not totally sure, but the ConsumerProgress does have a badger implementation. I'm guessing it was put there originally to align with the other db access interfaces.

I think some of the confusion might be because the interface and implementation naming. I'll update the namespaces to make it more clear